Makefile

```diff
@@ -103,3 +103,18 @@ learn-container:
 
 learn-push:
 	base/push-container docker.io/cockpit/learn
+
+amqp-shell:
+	$(DOCKER) run -ti --rm \
+		--publish=5671:5671 \
+		--volume=$(TASK_SECRETS):/run/webhook/secrets:ro \
```

Comment:
That looks wrong? should be `WEBHOOK_SECRETS`?

Reply:
yea I was a bit confused as that variable isn't defined in the Makefile.  think tasks-shell should mount those secrets too, but wasn't sure what the actual path should be in `/var/lib/cockpit-tasks/?`

Reply:
FTR, it's `/var/lib/cockpit-secrets/{release,tasks,webhook}/` now.